Updatet German

sjdo opened the pull request:
Fix Typos and some words

sjdo committed:
Updatet German

Fix Typos and some words

LightningMods commented:
iv checked with one of my german friends, and he said true and false is "wahr" and "falsch", are you german?

sjdo commented:
I'm full-blooded German, and it's true that True is Wahr and False is Falsch, but not for settings, there you use on and off.
And i need to change Dice form Würfel back to CD because I had a misconception there.

sjdo committed:
Dise Change and JRE Function Unknown

  The translation for "What more?" would be "Was mehr?" or Simple "Mehr", But that there is no function below I don't know exactly how to translate it

sjdo committed:
Remove 1 ##
